### ISSUE TITLE
Regression: ocamldoc lost unattached comment

### ISSUE THREAD

**Issue by vicuna** [ocaml/ocaml#6926]
**Original bug ID:** 6926
**Reporter:** bobot
**Status:** closed (set by @damiendoligez on 2015-07-21T14:46:58Z)
**Resolution:** fixed
**Priority:** high
**Severity:** major
**Version:** 4.02.2
**Target version:** 4.02.3+dev
**Fixed in version:** 4.02.3+dev
**Category:** ocamldoc
**Monitored by:** @dbuenzli @Yakobowski

## Bug description

`ocamldoc  -html toto.mli`, with toto.mli being:

```
(** ICI *)

type t = int

(** la *)

type t2 = int
```

produces in 4.02.1:

```
module Toto: sig .. end
ICI
type t = int 

la
type t2 = int 
```

and in 4.02.2:

```
module Toto: sig .. end
ICI
type t = int 

type t2 = int 
```

"la" disappeared

## Additional information

Doesn't append if the types are recursive and the comment enclosed inside them.

**Comment by vicuna** [comment]
**Comment author:** @dbuenzli

This is quite bad as it drops all sectional headings of the form (** {1 Heading} *). 

See e.g. https://github.com/dbuenzli/xmlm/issues/10

**Comment by vicuna** [comment]
**Comment author:** @damiendoligez

Fixed in branch 4.02 (rev 16229)
